deep4cast/forecasters.py

```diff
@@ -146,7 +142,7 @@ def optimizer(self, optimizer):
 
     def get_optimizer_args(self):
         """Get optimizer parameters."""
-        args = getargspec(self._optimizer.__class__)[0]
+        args = getargspec(self._optimizer.__class__)[0]  # REVIEW @Toby: getargspec is being deprecated
```

Comment:
@bischtob , this is a warning from PyCharm. It suggests using `inspect.signature` function.